ipaclient/install/ipa_certupdate.py

```diff
@@ -106,15 +106,31 @@ def run_with_args(api):
 
     server_fstore = sysrestore.FileStore(paths.SYSRESTORE)
     if server_fstore.has_files():
+        # look up CA servers before service restarts
+        resp = api.Command.server_role_find(
+            role_servrole=u'CA server',
+            status='enabled',
+        )
+        ca_servers = [server['server_server'] for server in resp['result']]
```

Comment:
This is also going to require that the current principal has read access to roles. Today this works:
```
kinit someuser
ipa-certupdate
```
but ca_servers will be an empty list and I don't know what impact that will have.

Reply:
Oh, I see it will skip things. Still perhaps not expected but at least it won't break anything.

Reply:
@rcritten yes, empty result is handled.  But also, unprivileged users and read serverrole objects with default ACLs.

Reply:
Yeah, I must have run ipa server-find instead or something to verify. So yeah, this is fine.